tools/blockmigrator/blockmigrator.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"go.uber.org/zap"
+
+	"github.com/iotexproject/iotex-core/blockchain/blockdao"
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/db"
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// migrateHeight is the blockchain height being migrated to
+var migrateHeight int
+
+// configPath is the config file path
+var configPath string
```

Comment:
I think you didn't add code to pass this var in from command line

Reply:
Fixed.